### PR TITLE
APIGOV-16095 - Create page in Connect AWS Gateway “deploy-your-agents-with-amplify-cli”

### PR DESCRIPTION
**Pull request by jcollins-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by jcollins-axway** [comment]
@dfeldick @sbolosan @kflan-io @vivekschauhan - will one of you validate for content, please

File: content/en/docs/central/connect-aws-gateway/deploy-your-agents-with-amplify-cli.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+### Step 1: Identify yourself to AMPLIFY platform with AMPLIFY CLI
+
+To use Central CLI to log in with your AMPLIFY Platform credentials, run the following command and use `apicentral` as the client identifier:
+
+```shell
+amplify auth login --client-id apicentral
+```
+
+A browser will automatically open.
+Enter your valid credentials (email address and password). Once the “Authorization Successful” message is displayed, you can go back to the AMPLIFY CLI.
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/deploy-your-agents-with-amplify-cli.md:40]: Maybe a hint that they can close that browser page.  They don't need that open anymore...

Comment (dfeldick) [reply content/en/docs/central/connect-aws-gateway/deploy-your-agents-with-amplify-cli.md:40]: like that

**Review by sbolosan — COMMENTED** [review]
Looks really good!!! Do we dare (for the section of the input) tell them or give them a hint as to what the regex is expecting?  maybe not all the prompts, but at least for the vpc- sg- and subnet- 

If its too confusing, then we can pass on that...

**Review by dfeldick — APPROVED** [review]

**Comment by jcollins-axway** [comment]
> Looks really good!!! Do we dare (for the section of the input) tell them or give them a hint as to what the regex is expecting? maybe not all the prompts, but at least for the vpc- sg- and subnet-
> 
> If its too confusing, then we can pass on that...

Added

**Review by sbolosan — APPROVED** [review]
Awesome!  Love the page!

**Review by kflan-io — APPROVED** [review]

**Review by lbadenhop — APPROVED** [review]

**Review by dfeldick — APPROVED** [review]

**Review by sbolosan — APPROVED** [review]

**Review by kflan-io — APPROVED** [review]

**Pull request merged**